api/types.proto

```diff
@@ -280,6 +280,23 @@ message ContainerStatus {
 
 	int32 pid = 2 [(gogoproto.customname) = "PID"];
 	int32 exit_code = 3;
+
+	HealthStatus health = 4;
+}
+
+// HealthStatus stores information about the container's healthcheck results.
+message HealthStatus {
+	string status = 1;
+	int32 failing_streak = 2;
```

Comment:
I'm a little bit confused about `failing_streak`. We have 4 options:

1. leave its value for engine to calculate.
2. go through all `HealthcheckResult` logs, and do calculation by our own.
3. remove this field, because now we don't use it.
4. **remove the whole `health` field in `ContainerStatus`**. Now we get health info from events api, instead of from `containerStatus`

Now we choose option 1. I don't understand the benefits of choice 2. Option 3 and 4 are also acceptable because currently we don't use this field.

I prefer to choose option 4, remove the whole `health` field in `ContainerStatus`. What's your opinion？ @stevvooe @nishanttotla